geocompr.bib

```diff
@@ -1,34 +1,37 @@
 @misc{_map_1993,
   title = {Map Projections},
-  year = {1993},
-  publisher = {US Geological Survey},
-  doi = {10.3133/70047422}
+  date = {1993},
+  doi = {10.3133/70047422},
+  url = {https://doi.org/10.3133/70047422},
+  organization = {US Geological Survey}
 }
 
 @book{abelson_structure_1996,
   title = {Structure and Interpretation of Computer Programs},
   author = {Abelson, Harold and Sussman, Gerald Jay and Sussman, Julie},
-  year = {1996},
+  date = {1996},
```

Comment:
@Robinlovelace -- I think this is a wrong export (I think that you used Better BibLatex, but should Better Bibtex, but I am not sure)

Reply:
yeah maybe, I noticed that. Will try again.

Reply:
Live update:

![image](https://github.com/user-attachments/assets/cad336e7-afd0-45c4-83d8-09cf005c3fc5)